src/client/client.ts

```diff
@@ -172,6 +175,27 @@ export class Client extends BaseClient {
     return this.fetchResources("passportScores", {}, passportScoreBuilder);
   }
 
+  getMatchingEstimates(
+    roundId: string,
+    chainId: string,
```

Comment:
As a reader,`chainId` being a string here makes me wonder whether it should be something like `"mainnet"` or `String(1)`.

Reply:
@bard it's true, it might be confusing! chain IDs are big numbers in the evm specs, what shall we use here? in theory now it's a string representation of the bigint

Reply:
A bigint if you want to allow querying for any possible chain, or an enum if supported chains are known ahead of time and you want to remove one possible source of errors as is done in Grants Stack.

Reply:
ah yeah it makes sense here if it's not even serialized/deserialized, I thought it was something we had stored somewhere